### ISSUE TITLE
Incremental update of Elastic Beanstalk environment exception during push command

### ISSUE THREAD

**Issue by Shinehah-Gnolaum** [aws/aws-toolkit-eclipse#151]
When I try to run a web project on a server using elastic beanstalk using incremental deployment, I received an error with this message in Details:

Unable to update environment with an incremental deployment
If you continue having problems with incremental deployments, try turning off incremental deployments in the server configuration editor.
Exception caught during execution of push command

**Comment by zhangzhx** [comment]
Can you try turning off the `incremental deployments` in the configuration editor and try again?

**Comment by Shinehah-Gnolaum** [comment]
@zhangzhx Yes. I have tried that, and it works, as long as I also don't check the box to save sessions in DynamoDB.
